Dockerfile

```diff
@@ -7,7 +7,7 @@
 #
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
-FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.0
+FROM quay.io/eclipse/che-sidecar-kubernetes-tooling:1.0.4
 
 ENV KAMEL_VERSION 1.0.0-M4
```

Comment:
Why not move up to 1.0.0-RC1 ? 

https://github.com/apache/camel-k/releases/tag/1.0.0-RC1

Reply:
the VS Code Camel K extension 0.0.10 was released and tested with 1.0.0-M4. it is using the 1.0.0-M4 by default. I prefer to stick with this version.